aten/src/ATen/functorch/BatchedTensorImpl.cpp

```diff
@@ -25,20 +29,24 @@ BatchedTensorImpl::BatchedTensorImpl(DispatchKeySet key_set, Tensor value, int64
 {
   TORCH_INTERNAL_ASSERT(value_.defined());
   set_storage_access_should_throw();
-  set_custom_sizes_strides(SizesStridesPolicy::CustomStrides);
+  set_custom_sizes_strides(
+      value_.is_nested() ? SizesStridesPolicy::CustomSizes : SizesStridesPolicy::CustomStrides);
```

Comment:
No, this doesn't work yet :)

> I'd expect us to provide a custom sizes implementation that raises an error (since the underlying thing is a NestedTensor)

you're exactly right about this, it's a TODO for me

Reply:
Update should address this